### ISSUE TITLE
Open source Ideal Graph Visualizer

### ISSUE THREAD

**Issue by fniephaus** [oracle/graal#4998]
### TL;DR

Open source [Ideal Graph Visualizer (IGV)](https://www.graalvm.org/22.2/tools/igv/) to make it easier for third-party compiler and language developers to use and contribute to it.

### Goals

IGV is currently only available through [OTN](https://www.oracle.com/downloads/graalvm-downloads.html) and released together with GraalVM EE. We plan to add the sources of IGV to the `oracle/graal` repository. This means that IGV releases can also be made available as part of GraalVM CE and CE dev builds. In addition, this will also allow third-party compiler and language developers to follow and develop against the latest GraalVM updates on the `master` branch.

### Non-Goals

- Turn IGV into an installable GraalVM component (requires additional work, but could be done)
- Merge with [the IGV fork maintained as part of the OpenJDK repo](https://github.com/openjdk/jdk/tree/master/src/utils/IdealGraphVisualizer)

**Comment by fniephaus** [comment]
https://github.com/oracle/graal/commit/6695488028c4a4498764a1c0f49af8f4b8ba3de5 has added the IGV sources.
